client/src/pages/MoviesList/index.js

```diff
@@ -1,18 +1,59 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 
 import { MovieListBackground } from '../../styles/Background'
 import Marquee from '../../components/Marquee'
 import Posters from './Posters'
 import SelectedMovie from './SelectedMovie'
+import TurnsTable from './TurnsTable'
+import { useHouses } from '../../providers/Houses'
+import { useTurns } from './utility'
+import { ReelImage } from '../../styles/LoadingReel'
 
 const MoviesList = () => {
   const [selectedMovie, setSelectedMovie] = useState()
 
+  const { currentHouse, userId } = useHouses()
+  const { getHouseTurns, houseTurns } = useTurns()
+
+  useEffect(() => {
+    if (currentHouse && !houseTurns) {
+      getHouseTurns(currentHouse.id)
+    }
+  }, [getHouseTurns, houseTurns, currentHouse])
+
+  if (!currentHouse || !houseTurns) {
+    return (
+      <MovieListBackground>
+        <ReelImage />
+      </MovieListBackground>
+    )
+  }
+
   return (
```

Comment:
This return statement is doing a lot. There's lot conditional renders occuring in a single return statement. 
In general, it's best to use several return statements rather than try to fit too many conditional operators in a single return.

When your return logic starts to look like this, it's a good signal that you might want to break the returns into different, smaller components. 

It looks like this component is handling
1. loading state
2. display turns table when it is **not** user's turn
3. Posters stack when it **IS** user's turn
4. Displaying success state when the user selects a movie

I'd argue that the PosterStack and Selected movie components could be housed in their own Parent component that handles the logic of whether to render PosterStack or SelectedMovie